@here/harp-mapview/lib/geometry/overlayOnElevation.ts

```diff
@@ -20,6 +29,58 @@ function overlayObject(object: TileObject, displacementMap: THREE.DataTexture):
     }
 }
 
+const worldCoords = new THREE.Vector3();
+
+/**
+ * Overlays the specified coordinates on top of elevation data if available.
+ *
+ * @param tileCoords World coordinates relative to the `tile` center.
```

Comment:
Please add something like: "tileCoords" will be modified by terrain height at that position.